catalog/ui/search-ui/search-endpoint/src/test/java/org/codice/ddf/ui/searchui/query/service/WorkspaceServiceTest.java

```diff
@@ -24,23 +21,9 @@
 
     @Test
     public void test() throws Exception {
```

Comment:
❗️ Now that we've removed all those commented out lines, I just realized that this test doesn't do anything anymore. We should do like we did for the WSF 1.0 test class and open a ticket to get this class fixed.